apps/give-cast/utils/receipt.ts

```
@@ -0,0 +1,13 @@
+import { postApi } from '@/utils/api';
+
+export default async function sendReceipt(data: any) {
+  console.log('Sending receipt...', data);
+  try {
+    const result = await postApi('receipt', data);
+    console.log('Result', result);
+    return { success: true, result };
+  } catch (ex: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)